HW_3/task_2.py

```diff
@@ -0,0 +1,3 @@
+variable = 2
+print(variable * 2, variable * variable)
+print(variable / 2, variable / variable)
```

Comment:
and one more solution is to use bit operators

print(variable >> 1)
print(variable << 1)